mmhuman3d/data/data_structures/human_data.py

```diff
@@ -30,39 +30,57 @@
     'bbox_xywh': {
         'type': np.ndarray,
         'shape': (-1, 5),
-        'temporal_dim': 0
+        'slice_dim': 0
     },
     'config': {
         'type': str,
+        'slice_dim': -1
     },
     'keypoints2d': {
         'type': np.ndarray,
         'shape': (-1, -1, 3),
-        'temporal_dim': 0
+        'slice_dim': 0
     },
     'keypoints3d': {
         'type': np.ndarray,
         'shape': (-1, -1, 4),
-        'temporal_dim': 0
+        'slice_dim': 0
     },
     'smpl': {
         'type': dict,
+        'slice_key': 'betas',
+        'slice_dim': 0
+    },
+    'smplh': {
+        'type': dict,
+        'slice_key': 'betas',
+        'slice_dim': 0
     },
     'smplx': {
         'type': dict,
+        'slice_key': 'betas',
+        'slice_dim': 0
     },
     'meta': {
         'type': dict,
     },
     'keypoints2d_mask': {
         'type': np.ndarray,
         'shape': (-1, ),
-        'temporal_dim': -1
+        'slice_dim': -1
+    },
+    'keypoints2d_convention': {
+        'type': str,
+        'slice_dim': -1
     },
     'keypoints3d_mask': {
         'type': np.ndarray,
         'shape': (-1, ),
-        'temporal_dim': -1
+        'slice_dim': -1
+    },
+    'keypoints3d_convention': {
```

Comment:
For custom attributes absent in _HumanData_SUPPORTED_KEYS, as there is no `major_dim` defined, we will treat dim 0 as a possible major dim. When the length of dim 0 equals to a well-defined major_dim, it will be sliced automatically.
Also, the user could update _HumanData_SUPPORTED_KEYS and re-set HumanData.SUPPORTED_KEYS to make it supported.